install-netkit-jh.sh

```diff
@@ -201,22 +206,21 @@ if [ "${PREVIOUS_INSTALL_FOUND}" = true ]; then
 	echo ""
 	echo "Restoring configuration from previous installation."
 	${TARGET_INSTALL_DIR}/setup_scripts/handle_config.sh "${BACKUP_INSTALL_DIR}" "${TARGET_INSTALL_DIR}"
-fi 
+fi
+
+echo "Netkit-JH should now be installed. Checking configuration..."
 
-# check netkit install now works
-# TODO: Get this to work... Netkit installs fine, but the check-configurator script doesn't read the environment variables properly so thinks something is wrong
-source ~/.bashrc
-cd "${TARGET_INSTALL_DIR}"
+# Ubuntu (and similar) distributions prevent .bashrc from running in a
+# non-interactive shell. So here we can just evaluate the environment variables
+# export commands
+eval "$NK_ENV_VARS"
 
-# ./check_configuration.sh
-# encourage user to set environment variables for the current bash terminal
-echo "" 
-echo "Future terminals that you launch will automatically get the netkit settings."
-echo "To make the netkit settings available in this terminal, run the following command:"
-echo "source ~/.bashrc"
+cd "${TARGET_INSTALL_DIR}/setup_scripts"
+./check_configuration.sh || exit 1
 
+# Encourage user to set environment variables for the current terminal
 echo ""
-echo "Run source ~/.bashrc, or open a new terminal, and then run ${NETKIT_HOME}/setup_scripts/check_configuration.sh to ensure your Netkit installation works!"
+echo "To use Netkit-JH now, open a new terminal window or run source ~/.bashrc"
```

Comment:
Can you include:
`run source ~/.bashrc (or run source ~/.zshrc if you are using zsh)